Documentation/install/azure/azure-private.md

```diff
@@ -0,0 +1,56 @@
+# Azure private clusters
+
+Use Tectonic Installer to create clusters within an existing private Azure network. In addition to the standard Azure requirements, private clusters also require:
+* An existing Virtual Network with a pre-configured and connected VPN link.
+* A DNS domain pre-registered into Azure DNS, or a 3rd party DNS server that supports DNS UPDATE operations.
+
+Once these requirements are met, follow the guide to [Install Tectonic on Azure with Terraform][azure-terraform].
+
+## Connected VPN link
+
+Private Azure networks require that an external VPN be available and running prior to running Tectonic Installer.
+
+For more information, see the Microsoft Azure document [Connect an on-premises network to Azure using a VPN gateway][connect-azure].
```

Comment:
We should make it apparent that this is neither the only nor the endorsed way to setup the VPN. 
In fact we're not testing with the particular method. Our tests use an OpenVPN connection to an OpenVPN Access Server that we deploy into the cluster VNet. Azure docs include an example of how to achieve that: https://azure.microsoft.com/en-us/resources/templates/openvpn-access-server-ubuntu/
It is up to the user to decide which approach to use according to their own requirements.

Reply:
If it's not the endorsed way, why are we describing it? Should this doc describe the method used for testing instead?